### PR TITLE
Finish Chinese doc for CCActionManager and CCActionPageTurn3D

### PR DESCRIPTION
**Pull request by pandamicro**

### CODE REVIEW

**Comment by pandamicro** [comment]
@andyque Please review

File: cocos/2d/CCActionManager.h [hunk 1]
```diff
@@ -53,6 +53,13 @@ struct _hashElement;
     - When you want to run an action where the target is different from a Node. 
     - When you want to pause / resume the actions.
  
+ * @~chinese ActionManager是一个单例类，管理的所有动作。
```
Comment (zilongshanren) [inline cocos/2d/CCActionManager.h]: "管理的所有动作“，应该是“管理所有动作”？

File: cocos/2d/CCActionManager.h [hunk 2]
```diff
@@ -53,6 +53,13 @@ struct _hashElement;
...
+ * 通常你不需要直接使用这个类。99%的情况下，你将使用Node的接口，它提供了更友好的封装
+ * 但也有一些情况下，你可能需要使用这个单例。
+ * 例子:
```
Comment (zilongshanren) [inline cocos/2d/CCActionManager.h]: 例子->“示例”

File: cocos/2d/CCActionManager.h [hunk 3]
```diff
@@ -71,98 +78,149 @@ class CC_DLL ActionManager : public Ref
...
      */
     void removeAllActions();
 
-    /** Removes all actions from a certain target.
+    /** @~english Removes all actions from a certain target.
      All the actions that belongs to the target will be removed.
      *
-     * @param target    A certain target.
+     * @~chinese 从一个特定的目标删除所有动作。
+     * 属于该目标的所有的行动将被删除。
```
Comment (zilongshanren) [inline cocos/2d/CCActionManager.h]: “行动”->"动作“

**Comment by zilongshanren** [comment]
@pandamicro  review done.

**Comment by pandamicro** [comment]
Fixed, please check

**Pull request merged**